todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksFragment.kt

```diff
@@ -23,17 +23,12 @@ import android.support.v4.app.Fragment
 import android.support.v4.content.ContextCompat
 import android.support.v7.widget.PopupMenu
 import android.util.Log
-import android.view.LayoutInflater
-import android.view.Menu
-import android.view.MenuInflater
-import android.view.MenuItem
-import android.view.View
-import android.view.ViewGroup
+import android.view.*
```

Comment:
Remove star imports by going to AS -> preferences -> code style settings -> kotlin -> imports tab. In the future, we should push style settings to the repo, but for now, if you see your AS settings changing anything style-related, please change your settings.